pyaerocom/test/test_colocation.py

```diff
@@ -8,45 +8,70 @@
 import pytest
 import numpy as np
 import numpy.testing as npt
+import iris
+from cf_units import Unit
 
 from pyaerocom.conftest import (TEST_RTOL, testdata_unavail)
 from pyaerocom.colocation import colocate_gridded_ungridded
 from pyaerocom.colocateddata import ColocatedData
+from pyaerocom import GriddedData
+from pyaerocom import helpers
 
 @testdata_unavail
 @pytest.mark.parametrize('addargs,ts_type,shape,obsmean,modmean',[
-    (dict(), 
+    (dict(),
      'monthly', (2,12,8), 0.315930,0.275671),
     (dict(var_ref_outlier_ranges={'od550aer':[0.1,0.5]},
           var_outlier_ranges={'od550aer':[0.1,0.2]}),
      'monthly', (2,12,8), 0.227333,0.275671),
-    (dict(apply_time_resampling_constraints=False), 
+    (dict(apply_time_resampling_constraints=False),
      'monthly', (2,12,8), 0.316924,0.275671),
-    (dict(filter_name='WORLD-wMOUNTAINS'), 
+    (dict(filter_name='WORLD-wMOUNTAINS'),
      'monthly', (2,12,11), 0.269707, 0.243861),
-    (dict(use_climatology_ref=True), 
+    (dict(use_climatology_ref=True),
      'monthly', (2,12,13), 0.302636, 0.234147)
     ])
-def test_colocate_gridded_ungridded(data_tm5, aeronetsunv3lev2_subset, 
+def test_colocate_gridded_ungridded(data_tm5, aeronetsunv3lev2_subset,
                                     addargs, ts_type, shape,
                                     obsmean, modmean):
-    coldata = colocate_gridded_ungridded(data_tm5, aeronetsunv3lev2_subset, 
+    coldata = colocate_gridded_ungridded(data_tm5, aeronetsunv3lev2_subset,
                                          **addargs)
-    
-    assert isinstance(coldata, ColocatedData)    
+
+    assert isinstance(coldata, ColocatedData)
     assert coldata.ts_type == ts_type
-    assert coldata.shape == shape 
-    
+    assert coldata.shape == shape
+
     means = [np.nanmean(coldata.data.data[0]),
              np.nanmean(coldata.data.data[1])]
-    
+
     npt.assert_allclose(means, [obsmean, modmean], rtol=TEST_RTOL)
-    
+
+@testdata_unavail
+def test_colocate_gridded_ungridded_nonglobal(aeronetsunv3lev2_subset):
```

Comment:
Consider simplifying initiation of dummy cube and GriddedData object and also to move iit outside of this method, e.g. into a fixture in conftest, that can then also be used by other tests.

Reply:
Yes, created an issue for this. I would like to think a bit about what would be useful in general before implementing it.
#131.